src/server/services/wms/qgswmsrenderer.h

```diff
@@ -136,6 +137,8 @@ namespace QgsWms
       // Init a map with nickname for layers' project
       void initNicknameLayers();
 
+      void initLayerGroupsRecursive( QgsLayerTreeGroup *group );
```

Comment:
Do you think that `group` could be const in this case? Even a const reference could be sufficient. Your call.